PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+from cryptography.fernet import Fernet
+from datetime import datetime
+from time import strftime
+from tkinter import Entry, Label, LabelFrame, Tk, Listbox, N, S, E, W, NSEW, Button, Radiobutton, Checkbutton, END, ANCHOR, LEFT, Scrollbar, Toplevel, VERTICAL, IntVar, StringVar, Text, TRUE, RAISED, FLAT, Frame, SOLID, font
+from tkinter import Menu
+from tkinter import messagebox
+from tkinter import ttk
+from tkinter import filedialog
+from docx import Document
+from xlutils.copy import copy
+from docx.enum.text import WD_PARAGRAPH_ALIGNMENT
+from docx.enum.text import WD_TAB_ALIGNMENT
+from docx.shared import Cm
+import xlrd
+from os import startfile, getlogin
+import pyperclip
+from PIL.ImageTk import PhotoImage, Image
+import webbrowser
+from distutils.version import LooseVersion
+from configparser import ConfigParser
+
+
+def patch_crypto_be_discovery():
+
+    """
+    Monkey patches cryptography's backend detection.
+    Objective: support pyinstaller freezing.
+    """
+
+    from cryptography.hazmat import backends
+
+    try:
+        from cryptography.hazmat.backends.commoncrypto.backend import backend as be_cc
+    except ImportError:
+        be_cc = None
+    try:
+        from cryptography.hazmat.backends.openssl.backend import backend as be_ossl
+    except ImportError:
+        be_ossl = None
+
+    backends._available_backends_list = [
+        be for be in (be_cc, be_ossl) if be is not None
+    ]
+
+patch_crypto_be_discovery()
+
+root = Tk()
+
+root.wm_minsize(923, 0)
+root.iconbitmap(r'bin\tubes.ico')
+
+versionnr = LooseVersion('0.5.9')
+
+Config = ConfigParser()
+
+def ConfigSectionMap(section):
+    Config.read('config.ini')
+    dict1 = {}
+    options = Config.options(section)
+    for option in options:
+        try:
+            dict1[option] = Config.get(section, option)
+        except:
+            print("exception on %s!" % option)
+            dict1[option] = None
+    return dict1
+
+def downloadupdate():
+    startfile('aktualizator.exe')
+    root.destroy()
+
+
+def checkupdate():
+    A = open('bin/adres.txt').read()
+    try:
+        x = open(A+r'/version.tub')
+        y = x.readline()
+        if 'version' in y:
+            _, z = y.split('=')
+        z1 = LooseVersion(z)
+        if z1 > versionnr:
+            p1 = messagebox.askyesno("Aktualizacja", "Znaleziono nową wersję programu.\nCzy chcesz ją teraz pobrać?", default="yes")
+            if p1:
+                downloadupdate()
+        else:
+            pass
+    except FileNotFoundError:
+        pass
+
+def wczytajoddzial():
+    try:
+        c = ConfigSectionMap('Uzytkownik')['oddzial']
```

Comment:
Jednoliterowe nazwy zmiennych obsysają bo nie wiadomo co to jest i do czego służy (ewentualnie jako np. iterator w pętli for czy jako współrzędne x,y,z) A tutejsze `c`, `d`, `e` nie są nigdzie dalej używane. Każdy zbędny kod powinien od razu wylatywać bo zaciemnia obraz całości.